src/ToWords.ts

```diff
@@ -38,8 +36,6 @@ export class ToWords {
   public getLocaleClass(): ConstructorOf<LocaleInterface> {
     /* eslint-disable @typescript-eslint/no-var-requires */
     switch (this.options.localeCode) {
-      case 'en-BD':
-        return enBd;
```

Comment:
@blue-predator These changes could be the reason for test cases failing for en-BD, en-GB and tr-TR.